api/reaction_groups.go

```diff
@@ -0,0 +1,52 @@
+package api
```

Comment:
I moved reaction groups to their own file since they will be used again for PR comments.